pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1197,7 +1197,7 @@ type GcsStorageProvider struct {
 	BucketAcl string `json:"bucketAcl,omitempty"`
 	// SecretName is the name of secret which stores the
 	// gcs service account credentials JSON.
-	SecretName string `json:"secretName"`
+	SecretName *string `json:"secretName,omitempty"`
```

Comment:
I think just a string is OK, no need to be a pointer.

Reply:
fixed